### ISSUE TITLE
Python: Implement the new 'table' syntax

### ISSUE THREAD

**Issue by HenrikBengtsson** [MetabolicEngineeringGroupCBMA/seguid#62]
I've implemented Suggestions #39 and #58 in R.  Here are some examples for predefined tables:

```r
seguid("ACGT", table = "{DNA}")
seguid("ACGU", table = "{RNA}")
seguid("ABVT", table = "{IUPAC}")
seguid("PQIT", table = "{protein}")
```

Some examples of custom tables:

```r
seguid("ACGT", table = "AT,TA,CG,GC")  ## DNA
seguid("ACGU", table = "AU,UA,CG,GC")  ## RNA
seguid("PQIT", table = "A,C,D,E,F,G,H,I,K,L,M,N,P,Q,R,S,T,V,W,Y")  ## protein
```

Examples that extend an existing table:

```r
seguid("ABVT", table = "{DNA},BV,VB,DH,HD,KM,MK,NN,SS,WW")  ## IUPAC
```

* [x] This should be implemented in Python

**Comment by HenrikBengtsson** [comment]
```python
dna="at,ta,cg,gc"
seguid("acgt", table = dna)
```

This approach works at the CLI too, e.g.

```sh
dna="at,ta,cg,gc"
python -m seguid --table="$dna" <<< "acgt"
```

which is why I proposed it.

**Comment by BjornFJohansson** [comment]
This was implemented [7081698](https://github.com/MetabolicEngineeringGroupCBMA/seguid/commit/70816986c28a9d278ec18636adee90ab102d0678).
